examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>can we print / visulaize the candidate_features object? something like candidate_features.head() ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>